hudi-common/src/main/java/org/apache/hudi/common/util/collection/CloseableMappingIterator.java

```diff
@@ -22,8 +22,8 @@
 
 import java.util.function.Function;
 
-// TODO java-doc
-public class CloseableMappingIterator<I, O> extends MappingIterator<I, O> implements ClosableIterator<O> {
+public class CloseableMappingIterator<I, O> extends MappingIterator<I, O>
```

Comment:
Can we add some document for this `CloseableMappingIterator` class?

Reply:
Can you help me understand what you want to have clarified in there?

Reply:
At least not to introduce unnecessary compile warnnings.

Reply:
Not sure i understand what warnings you're referring to

Reply:
A class without any document would cause the mvn building process to report a warning log.